--- a/feeds/kenzok8/natcap/Makefile
+++ b/feeds/kenzok8/natcap/Makefile
@@ -71,7 +71,7 @@ endef
 define Package/natcapd
   $(call Package/natcapd/Default)
   TITLE:=natcapd agent for natcap
-  DEPENDS += +libustream-ssl +uclient-fetch \
+  DEPENDS += +libustream-openssl +uclient-fetch \
 			 +lua-cjson \
 			 +ipset \
 			 +dnsmasq-full +@PACKAGE_dnsmasq_full_ipset \
--- a/feeds/packages/net/shadowsocks-libev/Makefile
+++ b/feeds/packages/net/shadowsocks-libev/Makefile
@@ -89,7 +89,7 @@ define Package/shadowsocks-libev-ss-rule
   SUBMENU:=Web Servers/Proxies
   TITLE:=shadowsocks-libev ss-rules
   URL:=https://github.com/shadowsocks/shadowsocks-libev
-  DEPENDS:=+firewall4 \
+  DEPENDS:=+firewall \
 	   +ip \
 	   +resolveip \
 	   +ucode \
